swssconfig/sample/00-copp.config.json

```diff
@@ -22,7 +22,7 @@
     {
         "COPP_TABLE:trap.group.arp": {
             "trap_ids": "arp_req,arp_resp,neigh_discovery",
-            "trap_action":"trap",
+            "trap_action":"copy",
```

Comment:
this crashes on Mellanox switches
please do not merge yet, will clarify with SAI

Reply:
I think this change creates a loop for ICMPv6 RS & NS packets in an L2 MC-LAG scenario as i documented here:
https://github.com/sonic-net/SONiC/issues/1253